docs/unified/msi/hcl.md

```diff
@@ -33,11 +33,17 @@ hardware.
 
     <center>
 
-    === "PRO Z690-A (WIFI) DDR4"
+    === "PRO Z690-A (WIFI) (DDR4)"
         | CPU Model | Dasharo version |  Source  |
         |:---------:|:----------------------:|:---------:|
+        | 12th Gen Intel(R) Core(TM) i9-12900K | v1.1.1 | Dasharo HCL Report |
```

Comment:
The order seems a bit random, maybe let's sort the list?

Reply:
the 12900K is at the top hence my comment, the rest of the list looks sorted indeed

Reply:
@mkopec, now I get your point: 6c6f67fe7bbb

Reply:
@mkopec ping